deploy/kubernetes/helm/che/templates/deployment.yaml

```diff
@@ -125,13 +125,22 @@ spec:
         imagePullPolicy: {{ .Values.cheImagePullPolicy }}
         securityContext:
           runAsUser: {{ .Values.global.securityContext.runAsUser }}
+{{- if not .Values.che.disableProbes }}
```

Comment:
When the service will start proxying traffic to that pod? As soon as it becomes Running but tomcat is still starting?

Reply:
Yes, which I think is ok in debug installations this is intended for.